tool.go

```diff
@@ -750,9 +750,31 @@ func runNode(script string, args []string, dir string, quiet bool) error {
 	}
 
 	if runtime.GOOS != "windows" {
-		allArgs = append(allArgs, "--stack_size=10000", script)
+		// For non-windows OS environments, we've seen issues with stack space
+		// limits causeing Go std library tests that are recursion-heavy to fail
```

Comment:
s/causeing/causing/

Reply:
"std library" sounds weird. Why shorten one word, but not the other? Let's not shorten any, it's not helpful here.

s/std/standard/